microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -14,22 +14,22 @@ ms.date: 02/17/2021
 
     1.  "Microsoft Edge Add-ons" means a Microsoft owned or operated platform, however named, through which Microsoft Edge Extensions may be offered to or acquired by Customers.
 
-1.  SUBMISSION, CERTIFICATION, AND DISTRIBUTION OF APPS.  Section 3 of the App Developer Agreement is hereby deleted and replaced by the following:
+1.  SUBMISSION, CERTIFICATION, AND DISTRIBUTION OF APPS.  Section 3 of the App Developer Agreement (SUBMISSION, EVALUATION, CERTIFICATION, AND DISTRIBUTION OF APPS) is hereby deleted and replaced by the following:
 
     1.  Submission.  You must submit to Microsoft each Extension that you wish to make available through Microsoft Edge Add-ons, including any updates to each Extension.  You are solely responsible and liable for the Extensions you submit.  You are responsible for supporting your extension.  Microsoft will retain (or destroy) all copies of the Extension and other materials you submit.  Microsoft will not return them, so you must maintain your own backup copies.
 
     1.  Updates to Extensions.  You may submit updates to Extensions for Certification and distribution through Microsoft Edge Add-ons.  Those updates are subject to all of the requirements of this Agreement.  You understand that end users may receive updates automatically.  You may not add any new functionality to your Extension via an update which requires written consent from the Customer without first providing notice to the Customer and obtaining any consents as may be required by law in the markets where you choose to distribute your Extension.
 
     1.  Initial Certification.  Microsoft will test each Extension (including any updates) you submit for compliance with this Agreement (which includes the applicable Certification Requirements) and other policies made available to you (if any).  Microsoft will not make any Extension available through Microsoft Edge Add-ons unless and until the Extension is Certified.
 
-1.  AFFIRMATIVE DECLINE NOT REQUIRED.  Notwithstanding Section 4(i) of the App Developer Agreement, you aren't required to affirmatively decline participation in Microsoft Store for Business and Microsoft Store for Education.  Your Extension will only be available through the Microsoft Edge Add-ons.
+1.  AFFIRMATIVE DECLINE NOT REQUIRED.  Notwithstanding Section 4(i) of the App Developer Agreement (License to Customer for Store Apps), you aren't required to affirmatively decline participation in Microsoft Store for Business and Microsoft Store for Education.  Your Extension will only be available through the Microsoft Edge Add-ons store.<!-- todo: remove list item or update it, b/c App Developer Agreement removed Microsoft Store for Business and Microsoft Store for Education.  confirm that the section meant is "License to Customer for Store Apps" -->
```

Comment:
remove list item or update it, b/c App Developer Agreement removed Microsoft Store for Business and Microsoft Store for Education. 

confirm that the section meant is "License to Customer for Store Apps"

https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/microsoft/final/en-us/store/legal-and-policy-documents/ada/MS-Store-ADA-v8.9-FINAL-EN.pdf --
![image](https://github.com/user-attachments/assets/894b109c-5701-4fc6-bc39-37d27a1f837f)

@Bhuvana-Priya @sivMSFT

Reply:
i'm going to delete this list item (for review), b/c it's entirely obsolete and no longer applies, and likely the 4(i) section was deleted (ambiguous b/c present article relies exclusively on section numbering - brittle, not durable - and doesn't back that up by stating section names)